ci/Dockerfile

```diff
@@ -1,22 +1,23 @@
-FROM luxonis/depthai-base:latest
+FROM python:3.9-buster
 
-RUN apt install wget
+RUN apt-get update && apt-get install -y wget build-essential cmake pkg-config libjpeg-dev libtiff5-dev libavcodec-dev libavformat-dev libswscale-dev libv4l-dev libxvidcore-dev libx264-dev libgtk2.0-dev libgtk-3-dev libatlas-base-dev gfortran git
+
+ADD ci/docker_dependencies.sh .
+# RUN wget http://docs.luxonis.com/install_dependencies.sh && chmod +x install_dependencies.sh && ./install_dependencies.sh
+RUN ./docker_dependencies.sh  # for development only, should be removed before merge
```

Comment:
Address before merge

Reply:
done, it's actually not for development only as it turned out